developers/contributor-guide/current/weaviate-contributors-guide/improving-docs.md

```diff
@@ -0,0 +1,220 @@
+  ```
+  ./setup.sh
+  ```
+* Build the site and make it available on your local server
+  ```
+  ./run.sh
+  ```
+* Browse [http://localhost:4000](http://localhost:4000) to view the website.
+
+## Basic Folder Structure of Documentation Site
+
+```
+├── .github/                            #contains GitHub template files for issues and pull requests
+├── _data/                              #contains site's data files
+├── _includes/                          #contains numerous templates like header, footer, navbar, etc
+│   ├── benchmarks/                     
+│   ├── code/                     
+├── _layouts/                           #contains layouts for documentation and posts
+├── _posts/                             #contains blog posts
+├── _python/                        
+├── developers/                         #contains documentation
+│   ├── assets/                         #contains clipboard functionality
+│   ├── contributor-guide/              #contains contributor guide for Weaviate
+│   ├── weaviate/                       #contains files and folder for product documentation 
+├── img/                                #contains all site images
+├── js/                                 #contains JS files for various functionalities
+├── resources/                          #contains additional site pages like gsoc, gsod
+├── .editorconfig
+├── .gitignore
+├── .travis.yml                         #for continuous integration and deployment
+├── 404.html                            #custom not found page.
+├── CONTRIBUTING.md
+├── CODE_OF_CONDUCT.md
+├── Gemfile                             #contains gem dependencies for the site.
+├── Gemfile.lock
+├── LICENSE
+├── README.md
+├── _config.yml                         #contains Jekyll settings for the site
+├── _config_dev.yml                     #for site build time improvement
+├── blog.md                             #main blog page of the site
+├── index.md                            #homepage of weaviate.io
+├── podcast.md                          #podcast page of the site
+├── product.md                          #product page of the site
+├── resources.md                        #resource page of the site
+├── robots.txt                          #tells search engine crawlers which URLs are accesible
+├── run.sh                              #script for previewing the site
+└── setup.sh                            #script for setting up the website
+```
+
+## Liquid syntax
+
+Jekyll uses the [Liquid](https://shopify.github.io/liquid/) templating language to process templates. To learn more about Liquid, check out the [official Liquid Documentation](https://shopify.github.io/liquid/).
+
+Here are a few basic syntax which we use in our code:
+
+* `if` condititional
+
+This executes the block of code only if the given condition is true. It is executed in the following manner:
+
+```
+Published on {{ page.date | date: '%B %d, %Y' }}
```

Comment:
Liquid tags are not showing up in code blocks. Rather it is being rendered on the site.

Reply:
Need help with this @sebawita